resources/lib/functions.py

```diff
@@ -653,7 +656,7 @@ def processDirectory(results, progress, params):
 
         # set the item name
         # override with name format string from request
-        if name_format is not None:
+        if (name_format is not None) and (item.get("Type") == "Episode"):
```

Comment:
I really wanted to have this expanded to have a name format for any type.
I know you need to do this due to the fast you have mixed item types in the response now but that now forces name_formats to be usable only on type episode.